raiden_webui/__init__.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+RAIDEN_WEBUI_PATH = os.path.join(os.path.abspath(__file__), 'ui')
```

Comment:
You need to keep the `dirname`, or else it'll do something like `.../__init__.py/ui`.
Change it to `RAIDEN_WEBUI_PATH = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'ui')`

Reply:
Oh ok, thanks for pointing out